packages/ra-core/src/auth/useRefreshIdentity.ts

```diff
@@ -0,0 +1,36 @@
+import { useCallback } from 'react';
```

Comment:
This hook is undocumented on purpose - I'm not sure it's better than documenting the react-query queryKey